test/expectations/expectations_test_runner.rb

```diff
@@ -91,6 +95,27 @@ def test_#{expectation_suffix}__#{test_name}__does_not_raise
           RB
         end
       end
+
+      TEST_YARP_FIXTURES.each do |path|
+        class_eval(<<~RB, __FILE__, __LINE__ + 1)
+          def test_#{expectation_suffix}__#{uniq_name_from_path(path)}__does_not_raise
+            @_path = "#{path}"
+            source = File.read(@_path)
+            run_expectations(source)
+          rescue RubyLsp::Requests::Support::InternalRuboCopError, RubyLsp::Requests::Formatting::Error
+            skip "Fixture requires a fix from Rubocop"
+          rescue NoMethodError => error
+            skip "Fixture requires a fix from SyntaxTree" if error.message.include?("SyntaxTree::MLHSParen")
```

Comment:
Can you share which scenario caused this one?

Also, we need to raise the error again here since `NoMethodError` could potentially be an issue in the LSP itself.

Reply:
`inlay_hints`, `document_symbol`, `document_highlight`, `document_link`, `code_lens` & `semantic_highlighting` running against: https://github.com/ruby/yarp/blob/main/test/fixtures/methods.txt

Its crash here: 
https://github.com/ruby-syntax-tree/syntax_tree/blob/main/lib/syntax_tree/with_scope.rb#L78-L89 

I can open an issue to report also.

Reply:
I tested with your fix and specifying the `main` branch in the `gemfile`, it's working, all tests should be OK after bumping the requirement. 👍  I keep an eye on it, and I'll re-run the CI on this PR after the official release.